tests/collector/helpers/test_collector_runtime.py

```diff
@@ -66,3 +69,32 @@ def test_collect_gc_metrics(self) -> None:
 
         self.helper._collect_gc_metrics(plugin_data[0], True)
         assert len(self.helper.previous["data"]["metrics"]["gc"]) == 6
+
+    def test_parse_ignored_endpoints(self) -> None:
+        test_pair = "service1:endpoint1,endpoint2"
+        response = parse_ignored_endpoints(test_pair)
+        assert response == ["service1.endpoint1", "service1.endpoint2"]
+
+        test_pair = "service1;service2"
+        response = parse_ignored_endpoints(test_pair)
+        assert response == ["service1", "service2"]
+
+        test_pair = "service1"
+        response = parse_ignored_endpoints(test_pair)
+        assert response == ["service1"]
+
+        test_pair = ";"
+        response = parse_ignored_endpoints(test_pair)
+        assert response == []
+
+        test_pair = "service1:endpoint1,endpoint2;;;service2:endpoint1;;"
+        response = parse_ignored_endpoints(test_pair)
+        assert response == [
+            "service1.endpoint1",
+            "service1.endpoint2",
+            "service2.endpoint1",
+        ]
+
+        test_pair = ""
+        response = parse_ignored_endpoints(test_pair)
+        assert response == []
```

Comment:
If the above suggestions about the methods' location are implemented, could you move these tests to the respective test files?